### PR TITLE
Snyk updates: react libs

### PR DESCRIPTION
**Pull request by jshaughn**
Prod:
- react, react-dom 16.9.0 -> 16.13.1
- react-redux 7.1.0 -> 7.2.0
- remove react-bootstrap (no longer used now that we are PF4 only)

Dev:
- react-scripts          3.3.0 ->  3.4.0

@pbajjuri20 Please perform a regression test.  Things looked  fine to me.
@aljesusg Found another lib we can remove :)

### CODE REVIEW

**Comment by mtho11** [comment]
@jshaughn looks like there is still some yarn.lock issues

File: package.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "proxy": "https://kiali-istio-system.apps.jshaughn-dev.devcluster.openshift.com",
```
[the remark this answers is not in the document]
Comment (jshaughn) [reply package.json:2]: whoops, good catch.

File: package.json [hunk 2]
```diff
@@ -121,7 +121,7 @@
     "node-sass": "4.12.0",
     "prettier": "1.16.4",
     "pretty-quick": "2.0.1",
-    "react-scripts": "3.3.0",
+    "react-scripts": "3.4.0",
```
[the remark this answers is not in the document]
Comment (jshaughn) [reply package.json:124]: ok, will update.

File: src/components/JaegerIntegration/JaegerResults/JaegerTraceTitle.tsx [hunk 3]
```diff
@@ -30,6 +30,7 @@ export class JaegerTraceTitle extends React.Component<JaegerScatterProps> {
                 href={this.props.onClickLink}
                 style={{ right: '130px', fontSize: '16px', position: 'absolute' }}
                 target={'_blank'}
+                rel="noopener noreferrer"
```
Comment (mtho11) [inline src/components/JaegerIntegration/JaegerResults/JaegerTraceTitle.tsx:33]: ???

Comment (jshaughn) [reply src/components/JaegerIntegration/JaegerResults/JaegerTraceTitle.tsx:33]: This was  security warning that was actually failing CI.  Apparently you need this when using target={'_blank'}.

**Comment by jshaughn** [comment]
@mtho11 OK, thanks!  @pbajjuri20 it's ready for QE review when you have time.

**Comment by mtho11** [comment]
@jshaughn you might add something about the `children` and `ref` changes as those we weird little changes.

**Comment by jshaughn** [comment]
@mtho11 I will in the commit comment

**Comment by pbajjuri20** [comment]
Tested and passed on Kiali UI 1.18.0 (d25d844a899d371df796c58f314245ae398add5e)

**Pull request merged**